frontend/packages/schema-model/src/lib/mutations/convert-node.ts

```diff
@@ -55,3 +62,37 @@ export const convertPropToType = (uiSchemaNodes: UiSchemaNodes, pointer: string)
     })
   );
 };
+
+export const convertRefToField = (uiSchemaNodes: UiSchemaNodes, pointer: string) => {
+  const uiNodeIndex = getNodeIndexByPointer(uiSchemaNodes, pointer);
+  if (uiNodeIndex === undefined) {
+    throw new Error(`Pointer ${pointer}, can't be found.`);
+  }
```

Comment:
Godt poeng. I utgangspunktet skal det ikke være mulig at dette skjer i det hele tatt, og sånn sett er det jo greit, men erfaringen har jo vist gang på gang at disse situasjonene oppstår for brukerne likevel. Da må vi også kunne brukeren en bedre opplevelse, gjerne med en synlig feilmelding som kan rapporteres tilbake til oss. Problemet er at vi da i resten av koden også må ta høyde for at funksjonen kan returnere `undefined`.

Reply:
```
try {  
  const converted = convertRefToField(schemaNodes, pointer);
  // Further code that can crash
} catch (error) {
   alertService.openSnackbar('Feilmelding til bruker');
}
```

Is a lot easier to work with and debug... and even easier to test. Throwing exceptions is somewhat old fashion and that is why a lot of practitioners try to avoid this. Imho this is also a codesmell.

So I am back to throwing errors at points where input-data is bad.